content/docs/guides/traffic_management/circuit_breaking.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Circuit Breaking"
+description: "Using Circuit breaking to limit connections and requests"
+type: docs
+weight: 11
+---
+
+# Circuit Breaking
+
+Circuit breaking is a critical component of distributed systems and an important resiliency pattern. Circuit breaking allows applications to fail quickly and apply back pressure downstream as soon as possible, thereby providing the means to limit the impact of failures across the system. This guide describes how circuit breaking can be configured in OSM.
+
+## Configuring circuit breaking
+
+OSM leverages its [UpstreamTrafficSetting API][1] to configure circuit breaking attributes for traffic directed to an upstream service. We use the term `upstream service` to refer to a service that receives connections and requests from clients and return responses. The specification enables configuring circuit breaking attributes for an upstream service at the connection and request level. OSM leverages [Envoy's circuit breaking functionality](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/circuit_breaking) to implement circuit breaking for applications participating in the mesh.
+
+Each `UpstreamTrafficSetting` configuration targets an upstream host defined by the `spec.host` field. For a Kubernetes service `my-svc` in the namespace `my-namespace`, the `UpstreamTrafficSetting` resource must be created in the namespace `my-namespace`, and `spec.host` must be an FQDN of the form `my-svc.my-namespace.svc.cluster.local`. When specified as a match in an [Egress policy](/docs/api_reference/policy/v1alpha1/#policy.openservicemesh.io/v1alpha1.EgressSpec), `spec.host` must correspond to the host specified in the Egress policy and the `UpstreamTrafficSetting` configuration must belong to the same namespace as the `Egress` resource.
+
+Circuit breaking is applicable at both the TCP and HTTP level, and can be configured using the `connectionSettings` attribute in the `UpstreamTrafficSetting` resource. TCP traffic settings apply to both TCP and HTTP traffic, while HTTP settings only apply to HTTP traffic.
+
+The following circuit breaking configurations are supported:
+
+- `Maximum connections`: The maximum number of connections that a client is allowed to establish to all backends belonging to the upstream host specified via the `spec.host` field in the `UpstreamTrafficSetting` configuration. This setting can be configured using the `tcp.maxConnections` field and is applicable to both TCP and HTTP traffic. If not specified, the default is `2147483647` (2^32 - 1).
```

Comment:
```suggestion
- `Maximum connections`: The maximum number of connections that a client is allowed to establish to all backends belonging to the upstream host specified via the `spec.host` field in the `UpstreamTrafficSetting` configuration. This setting can be configured using the `tcp.maxConnections` field and is applicable to both TCP and HTTP traffic. If not specified, the default is `4294967295` (2^32 - 1).
```

Reply:
That's an oversight on my part. I'll need to fix this in the Go struct comments as well in the osm repo. I've fixed this everywhere the max uint32 value is referenced in this change.

Reply:
Corresponding change in osm: https://github.com/openservicemesh/osm/pull/4605